controllers/runner_controller.go

```diff
@@ -564,6 +579,140 @@ func mutatePod(pod *corev1.Pod, token string) *corev1.Pod {
 	return updated
 }
 
+func (r *RunnerReconciler) createJobRunnerRoleIfNotExist(ctx context.Context, runner *v1alpha1.Runner) error {
```

Comment:
Is there going to be a way to clean these up once all runners are deleted from a namespace?

Reply:
I can list runners in current namespace and remove the role if we just remove the last one